v2/printings/parhelion.json

```diff
@@ -250,11 +250,35 @@
   {
     "card_id": "matryoshka",
     "card_set_id": "parhelion",
-    "flavor": "{Card 6} Look at the little baby, even she helps!\n{Card 5} Sheʼs little—but fierce.\n{Card 4} Just as brave as his sisters.\n{Card 3} Sheʼs a typical middle child, really.\n{Card 2} Always there to back up her big sister.\n{Card 1} She leads and they all follow.",
+    "flavor": "She leads and they all follow.",
     "id": "33094",
     "illustrator": "Ed Mattinian",
     "position": 94,
-    "quantity": 6
+    "quantity": 6,
+    "layout_id": "copy",
+    "copy_quantity": 1,
```

Comment:
how about dropping this field out here, but keeping "quantity" in the sides?  You can still sum the quantities up for the test, but move *all* instances of the field into sides where they vary, rather than treating one differently.

wdyt?

We are going to need to do some formatting shenanigans for these cards either way.

Reply:
Technically the base card is also a copy. I know it's sort of weird to have 6 equal cards have one arbitrarily denoted to dominant version, but see my comment below.